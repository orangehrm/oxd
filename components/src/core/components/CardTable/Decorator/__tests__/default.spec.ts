import {mount} from '@vue/test-utils';
import DefaultDecorator from '@orangehrm/oxd/core/components/CardTable/Decorator/Default.vue';

const GLOBAL = {
  provide: {
    tableProps() {
      return {
        items: [],
      };
    },
  },
};

describe('CardTable > Decorator > Default.vue', () => {
  it('should renders OXD CardTable > Decorator > Default', () => {
    const wrapper = mount(DefaultDecorator, {
      global: GLOBAL,
      slots: {default: 'Test'},
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
