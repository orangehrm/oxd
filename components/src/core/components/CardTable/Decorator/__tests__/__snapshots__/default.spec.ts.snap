// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardTable > Decorator > Default.vue should renders OXD CardTable > Decorator > Default 1`] = `"Test"`;
